packages/components/src/ListView/Items/Items.component.js

```diff
@@ -0,0 +1,168 @@
+import React, { PropTypes } from 'react';
+import classNames from 'classnames';
+import { AutoSizer, List } from 'react-virtualized';
+
+import Item from './Item/Item.component';
+import theme from './Items.scss';
+
+function listClasses() {
+	return classNames(
+		theme['tc-list-items'],
+		'tc-list-items',
+	);
+}
+
+function itemsClasses() {
+	return classNames(
+		theme['tc-listview-items'],
+		'tc-listview-items',
+	);
+}
+
+function itemContainer(additionalClassName) {
+	return classNames(
+		theme['tc-item-container'],
+		theme[additionalClassName],
+		'tc-item-container',
+		additionalClassName,
+	);
+}
+
+
+class Items extends React.PureComponent {
+	constructor(props) {
+		super(props);
+
+		this.renderItem = this.renderItem.bind(this);
+		this.getRowHeight = this.getRowHeight.bind(this);
+		this.rowRenderer = this.rowRenderer.bind(this);
+		this.renderToggleAllOrItem = this.renderToggleAllOrItem.bind(this);
+		this.renderToggleAll = this.renderToggleAll.bind(this);
+
+		this.hasToggleAll = this.props.items.length > 1;
+	}
+
+	getRowHeight({ index }) {
+		if (this.hasToggleAll && index === 0) {
+			return 40;
+		}
+		return this.props.getItemHeight();
+	}
+
+	getRowCount() {
+		const { items } = this.props;
+		return this.hasToggleAll ? items.length + 1 : items.length;
+	}
```

Comment:
This could be a simple if statement:
```javascript
getRowCount() {
  if (this.hasToggleAll) {
    return this.props.items.length + 1;
  }
  return this.props.items.length;
}
```